libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/UrlUtils.java

```diff
@@ -271,4 +271,12 @@ public static String extractSubDomain(String domain) {
         }
         return "";
     }
+
+    public static String cleanupInputAddress(String siteAddress) {
```

Comment:
> Should this method be added to the UrlUtils class  in the WordPress-Utils-Android repository?

We use `git subtree` to manage that repo so, we just need to push at some point.

> Also, should the method be named something more descriptive like removeXmlRpcPhpSuffixFromSiteAddress (but maybe something a little shorter)? I don't think cleanupInputAddress is very intuitive.

Renamed it to `removeXmlrpcSuffix` with 2641755. WDYT?